openssh.yaml

```diff
@@ -122,11 +122,16 @@ subpackages:
     pipeline:
       - runs: |
           mkdir -p "${{targets.subpkgdir}}"/usr/sbin "${{targets.subpkgdir}}"/var/empty
+          chmod 755 "${{targets.subpkgdir}}"/var/empty
           mv "${{targets.destdir}}"/usr/sbin/sshd "${{targets.subpkgdir}}"/usr/sbin/
     dependencies:
       runtime:
         - openssh-keygen
         - openssh-server-config
+    scriptlets:
+      post-install: |
+        #!/bin/sh
+        ssh-keygen -q -N "" -t rsa -b 4096 -f /etc/ssh/ssh_host_rsa_key
```

Comment:
Why not ssh-keygen -A to generate all of the host keys at once?

Reply:
Good call!  I just  pushed that change.